src/commands/autocomplete/script.js

```diff
@@ -48,8 +49,9 @@ compinit;`)
       //   fs.copySync(cli, client)
       // }
       const plugins = await new Plugins(out).list()
-      const cmds = plugins.map(p => p.commands.filter(c => !c.hidden).map(c => {
+      const cmds = plugins.map(p => p.commands.filter(c => !c.hidden || !!c.id).map(c => {
         let publicFlags = Object.keys(c.flags).filter(flag => !c.flags[flag].hidden).map(flag => `--${flag}`).join(' ')
+        if (c.args && c.args.find(a => a.name === 'app')) publicFlags.concat(' --app')
```

Comment:
I don't understand. If it takes an app as an arg you add the app flag? Is this supposed to add functionality for args as flags?